### DISCUSSION TITLE
MailPoet marketing integration

### DISCUSSION THREAD

**Discussion by johnfisherman** [Codeinwp/otter-blocks#1926] [Ideas & Feedback]
I'd love to have a [MailPoet](https://www.mailpoet.com/) integration, much like the Mailchimp one you guys already have. So when a customer gets in touch via the contact form (an Otter block), their email automatically gets added to my MailPoet DB. 

I'm running Otter Pro and I'm using MailPoet as a WP plugin.

**Comment by HardeepAsrani** [comment]
Hey @johnfisherman, that's a good idea. We will look into it once we're done with current release cycle for the next one.
